raiden/tests/benchmark/_codespeed.py

```diff
@@ -1,30 +1,29 @@
 import json
-import socket
-import urllib.error
-import urllib.parse
-import urllib.request
+import os
+
+import requests
+
```

Comment:
ah, [I see now](https://github.com/raiden-network/raiden/pull/7016/files#r630983402)

how about making this a
```python
try:
    ...
except AttributeError:
    print("Codespeed environment variables not available, posting results would fail")
```

Reply:
Done (exception type changed to `KeyError`, though).